### ISSUE TITLE
The links Isis and isis study landing pages don't have enough contrast 

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/datagateway#901]
**Description**:

## Dark mode 
The Investigation visit id/dataset and hyperlinks (DOI in citation and on the side) doesn't have enough contrast to satisfy WCAG 2 AA. Suggested foreground colour change to #86b4ff.

The link in the header also it does not have enough contrast for both Light and Dark mode 

![Landing page(dark mode)](https://user-images.githubusercontent.com/83226114/139681027-53f569fa-ce91-472b-a473-4d4bce49b02a.png)

**Acceptance criteria**:
- [ ] A checklist
- [ ] of criteria to accept this story as done

**Comment by joelvdavies** [comment]
For the header link colour, there are two main options, one is making the header darker:
![image](https://user-images.githubusercontent.com/90245114/140058471-71f50d65-3148-4129-a086-9dec255214af.png)
(This uses `theme.palette.primary.main` for the background and `'#FF7E00'` for the link)
The other is using a link much closer to white e.g.
![image](https://user-images.githubusercontent.com/90245114/140059057-5ef3997d-e145-4af2-8973-05edcaffe3a4.png)
(This only changes the link colour to `'#FFCEA0'`)
